src/turndown.js

```diff
@@ -237,6 +237,37 @@ turndownService.addRule("getpageshortcode", {
   }
 })
 
+/**
+ * Build quote element shortcodes for instructor insights sections
+ **/
+turndownService.addRule("quoteshortcode", {
+  filter: (node, options) => {
+    if (node.nodeName === "DIV" && node.getAttribute("class")) {
+      if (node.getAttribute("class").includes("pullquote")) {
+        return true
+      }
+    }
+    return false
+  },
+  replacement: (content, node, options) => {
+    try {
```

Comment:
Is there any error in particular we're trying to capture? It looks like many of the other rules `replacement` don't have a similar try/catch block

Reply:
So I had this in there to debug the output while developing this, but I think there is value in having a generic try catch on turndown replacement functions.  Any uncaught errors from running HTML through turndown do not produce an error in the calling code.  I just tested trying to manually throw an exception with `throw` in a turndown rule and it was swallowed.  We could ask the author about why it behaves that way, but I have a feeling that due to the chaotic nature of processing raw HTML into markdown errors are suppressed.  A generic try catch that logs the error into a file seems like a good workaround for now.  What do you think?